content/en/agent/faq/agent-v6-changes.md

```diff
@@ -0,0 +1,564 @@
+| `proxy_user`                 | Superseded by `proxy`, see the [proxy][3] documentation for details.                                                  |
+| `proxy_password`             | Superseded by `proxy`, see the [proxy][3] documentation for details.                                                  |
+| `proxy_forbid_method_switch` | Obsolete                                                                                                              |
+| `use_mount`                  | Deprecated at the Agent-level and moved to the [Disk check][4].                                                       |
+| `device_blacklist_re`        | Deprecated at the Agent-level and moved to the [Disk check][4] as `device_blacklist`.                                 |
+| `use_curl_http_client`       | Obsolete                                                                                                              |
+| `exclude_process_args`       | Deprecated feature                                                                                                    |
+| `check_timings`              | Superseded by internal stats                                                                                          |
+| `non_local_traffic`          | Superseded by `dogstatsd_non_local_traffic` for Dogstatsd and `apm_config.apm_non_local_traffic` for the Trace Agent. |
+| `dogstatsd_target`           |                                                                                                                       |
+| `dogstreams`                 | Deprecated feature, use the [Logs Agent][5] instead.                                                                  |
+| `custom_emitters`            |                                                                                                                       |
+| `forwarder_log_file`         | Superseded by `log_file`                                                                                              |
+| `dogstatsd_log_file`         | Superseded by `log_file`                                                                                              |
+| `jmxfetch_log_file`          | Superseded by `log_file`                                                                                              |
+| `syslog_port`                | Superseded by `syslog_uri`                                                                                            |
+| `check_freq`                 |                                                                                                                       |
+| `collect_orchestrator_tags`  | Implemented in metadata collectors                                                                                    |
+| `utf8_decoding`              |                                                                                                                       |
```

Comment:
Same question, should we just say they are now `Deprecated feature` ?

Reply:
It's in the removed list but there were no notes about it on current doc